llvm/tools/sycl-post-link/ModuleSplitter.cpp

```diff
@@ -310,10 +339,34 @@ ModuleDesc extractSubModule(const ModuleDesc &MD,
 // points that are specified in ModuleEntryPoints vector.
```

Comment:
// The function produces a copy of input LLVM IR module M with only those functions and globals that can be called 
// from entry points that are specified in ModuleEntryPoints vector.